README.md

```diff
@@ -1,2 +1,2 @@
 # TextOrVideoSummarizer-G18-PS24
-Repo for Text/Video summarizer
+Repo for Text/Video summarizer  
```

Comment:
No instructions in README on how to run your code? README must be elaborate enough to explain your project and give step by step guide for others to run your code.